docs/resources/domain_delegation.md

```diff
@@ -9,7 +9,7 @@ Provides a DNSimple domain delegation resource.
 This resource allows you to control the delegation records (name servers) for a domain.
 
 ~> **Note:** This resource currently only supports the management of domains that are registered with DNSimple.
--> **Note:** When this resource is destroyed, only the Terraform state is removed; the domain delegation is left intact and unmanaged by Terraform.
+~> **Note:** When this resource is destroyed, only the Terraform state is removed; the domain delegation is left intact and unmanaged by Terraform.
```

Comment:
There is no need for this change. The issue comes from the fact that there is no new line separating the notices. You can use [Terraform Doc Preview Tool](https://registry.terraform.io/tools/doc-preview) to check out how it will be rendered.

w/ space
<img width="724" alt="image" src="https://user-images.githubusercontent.com/17004703/234481083-99ecc084-45bb-4721-9c13-b3911f223aa3.png">

Reply:
So we want to keep the second note as a blue info? Both are titled *Note*; if one is not the same priority we should make the yellow box say *Warning* or something.